doc/Database.md

```diff
@@ -55,12 +55,13 @@ This table should be removed in favor of querying `validator` directly.
 
 ### `accumulated_fee` table
 
-Note sure why we need this.
+Keep track of fees as blocks are externalized (when the UTXO set has all the inputs still)
 
-| Field name | SQL Type | D type     | Attributes  | Comment             |
-|------------|----------|------------|-------------|---------------------|
-| public_key | TEXT     | PublicKey  | PRIMARY KEY |                     |
-| fee        | TEXT     | Enrollment |             | Should be `INTEGER` |
+| Field name      | SQL Type | D type     | Attributes  | Comment             |
+|-----------------|----------|------------|-------------|---------------------|
+| public_key      | TEXT     | PublicKey  | PRIMARY KEY |                     |
+| fee             | TEXT     | Enrollment |             | Should be `INTEGER` |
```

Comment:
Is the D type correct ??

Reply:
I'll add a commit to fix. It was wrong before.

Reply:
Updated